src/commands.js

```diff
@@ -99,10 +99,12 @@ export const mine = msg => {
 
 // Sends a DM with the help embed dialog
 export const help = msg => {
-  const description =
-    'You can find a list of commands here:' +
-    '\nhttps://sirmerr.github.io/camperbot/#/camperbot/commands' +
-    '\n\nFor a specific command help, use `[!h|!help] CommandName` (for example `!h !marshmallow`)';
+  const description = `You can find a list of commands here:
+       https://sirmerr.github.io/camperbot/#/camperbot/commands
+       \nFor a specific command help, use \`[!h|!help] CommandName\` (for example \`!h !marshmallow\`)`.replace(
+    /^ +/gm,
```

Comment:
Since you use `.replace(/^ +/gm, '')` a lot, you can make a utility function which does just that to a string to cut back on this type of repetition. For example:

```js
const removeSpaces(text) => {
    return text/replace(/^ +/gm, '')
}
```

```js
const description = removeSpaces(`You can find ...`);
```